CHANGELOG_PENDING.md

```diff
@@ -1,4 +1,7 @@
 ### Improvements
 
+- [sdk/go] Go SDK now properly outputs concise diagnostic error logs
```

Comment:
This is just a bit imprecise; the change affects just the failing Go programs right? Perhaps something like "Failing Go programs now display only one error message in Pulumi CLI"